### DISCUSSION TITLE
Alpha ratio validation additional plots

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#35] [Comments_HIG_23_001] [answered]
> Validation region:
>> - Do you make any statistical test that the prediction and the validation region data agree?
>> - Could you also validate in the 110-135 GeV mass region (no need to worry about a Higgs peak for validation)?
>> - Can you please also add validation results for 2016 and 2017 data?

- [ ] Validate the alpha ratio method also in region 110-135 GeV.
- [ ] also add validation results for 2016 and 2017 data

**Comment by jialin-guo1** [comment]
@ram1123 
I firstly add comparisons of Zjet from MC and from Alpha for you review 

merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/c86294bd-85a0-428c-b9cb-63484854235e)

resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/22c86ddd-9572-46ee-938e-8dc41970282d)

**Reply by ram1123** [reply]
This is good but can you just plot the two lines instead of adding on the stack plot.

This will ensure that there is no mistake in the plotting.

**Reply by jialin-guo1** [reply]
merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/cd224e08-fc76-4d45-a91d-aab28319400a)

resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/e83f11f4-3ef8-4750-a2cd-3287647b4d83)

**Reply by ram1123** [reply]
For the resolved case, now the distribution does not match. So, in the stack plot that you added previously had an issue. Right?

**Reply by jialin-guo1** [reply]
No they are the same. I would like to say that is not an issue. because the number of Zjet background is much larger than TT,WW and ZV. thus even I stack them, it will be hard to see much difference in log scale. i.e (N(Zjet)=10702) >> (N(ZV)=558)

**Reply by ram1123** [reply]
Then why does the merged case shows a large difference when you added it to the stack?

**Comment by jialin-guo1** [comment]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/164e8e44-7adb-4482-8a28-272db3976299)

Hi @ram1123 I fixed error bar issue. it because of a bug in my plotting code. now I update with Validation Region(110-135)
2018 resolved Validation in 110-135 region

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/4971f8e9-ad0d-4f73-9a20-981670fa23fb)

2017 resolved Validation in 110-135 region

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/73021abe-54be-4317-87f3-cde2e4b4f246)

**2016postAPV resolved Validation in 110-135 region**

![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/a466f7d3-e4dd-4824-87e7-3f8cc3e3aba5)

**2016preAPV resolved Validation in 110-135 region**

**Comment by jialin-guo1** [comment]
**Shape in region 70< & >110 in merged case**

![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/547e3fa1-decd-473e-9edf-ba874f0bb8a4)

**Reply by ram1123** [reply]
Good just do the same for resolved and we are done.

**Comment by ram1123** [comment]
# Task-1: Check assumption

Also, we need to get the comparison plots for the kinematics for the mZZ in the signal and sideband region.

The list of variables that we need for ZZ system are:

- [ ] $m_{ZZ}$
- [ ] $p_T$
- [ ] $\eta$
- [ ] $\phi$

# Task-2: Changing the VR and SB region

We also need to re-do alpha method with updated definition of the validation region and side-band region

Side-band region: 40 - 70, 110 - 135, 165 - 180
Validation region: 135 - 165

# Task - 3: Correlation check

Plot a 2D plot for MZZ and MJ separately in 

- Signal region
- Side-band region

Using the signal sample.

**Reply by jialin-guo1** [reply, marked as answer]
# Task - 3: Correlation check

Plot a 2D plot for MZZ and MJ separately 

| Signale Region | Side-band Region |
|---|---|
| ![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/09dac355-d7e5-4bfb-a37d-f78a3a885bef)| ![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/26dd6c15-4b7e-4b75-a388-c4259387d517) | 


# Task - 1: Check assumption
| massZZ | pt | Eat | Phi |
|---|---|---|---|
|![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/2beb305a-96e7-429d-a0ae-2c3f04fb2dbf)|![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/af6c5ad8-e55b-4513-86cb-9dc8f1423cce)|![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/317b9a2e-9098-4871-a1a7-c123a35cbc31)|![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/a6b28406-5db3-4ce2-bbb6-18447a9cbb80)|

# Task -2
https://github.com/ram1123/2l2Q_limitSettingTool/discussions/35#discussioncomment-6754879

**Comment by jialin-guo1** [comment]
I update new validation plots for all three year here
Side-band region: 40 - 70, 110 - 135, 165 - 180
Validation region: 135 - 165

2018
merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/29b6aa78-7d03-44ff-92f4-7149e6e0e1da)
resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/5575c688-f74a-4b29-b35e-39b80f5e2ba2)

2017
merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/9fc956b7-d10e-4368-bce6-9a85dfcd3841)
resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1ce0971e-33d0-4a4e-8c6f-cbe802c16669)


2016
merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/290ab877-1de8-4c49-b6e8-2d0fc32deff0)
resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/86d29f79-fbe5-407e-a4f4-ea5026bf1b91)

**Comment by jialin-guo1** [comment]
BTW part of SB is 105-135 because our SR is 70-105GeV
